client/src/app/_services/auth.service.ts

```diff
@@ -125,13 +125,15 @@ export class AuthService {
                         createdAt: connectedUser.barman.createdAt,
                         barman: connectedUser.barman,
                     }));
+                    this.ngxRolesService.addRole('BARMAN', ['']);
                 } else if (connectedUser.specialAccount) {
                     this.$currentUser.next(new ConnectedUser({
                         accountType: 'SpecialAccount',
                         username: connectedUser.specialAccount.connection.username,
                         createdAt: connectedUser.specialAccount.createdAt,
                         specialAccount: connectedUser.specialAccount,
                     }));
+                    this.ngxRolesService.addRole('SPECIAL_ACCOUNT', ['']);
```

Comment:
If you add the same role twice, what does it do?

Reply:
I don't know, but why are you asking ?

Reply:
Okay nevermind, but why creating a role with empty permissions ?

Reply:
It's just a way to identify the connected account type as a role. 
As a result, I can use it like a role in the menu component.

Reply:
Okay right